_usage/mvu.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: Performing a Major Version Upgrade
+nav_order: 9
+---
+
+## Performing a major version upgrade
+
+Babelfish supports major version upgrade (MVU) from implementations deployed with a Babelfish PostgreSQL server version 13.6 or later. Currently the Babelfish PostgreSQL server only supports an upgrade path using the `pg_upgrade` module. During the upgrade, you will need to:
+
+- Perform a major version upgrade of the PostgreSQL server.
+- Update the Babelfish extensions to the target version.
+- Confirm that the Babelfish specific GUCs are the same in the source and target versions.
+
+Babelfish supports a major version upgrade on Babelfish PostgreSQL servers version 13.6 or later. You can use the following release as an upgrade source (the server from which you'll upgrade):
+
+- Babelfish 1.3.1, built with PostgreSQL 13.7
+
+The target server must be running Babelfish PostgreSQL servers, version 14.5 or later.  You can use the following release as an upgrade target (the version to which you'll upgrade):
+
+- Babelfish 2.2.0, built with PostgreSQL 14.5
+
```

Comment:
What happens when the versions involved are not conforming to the above? Will APG tell the user? Will there be an error? (immediately? after some time?) How can the user find out that an upgrade failed, and whether it was due to the versions not being supported?

Reply:
@rishabhtanwar29 @surendravishnoi  

Can you comment on the questions please?

Reply:
as of now behaviour is undefined on OSS side. APG has managed experience with errors given immediately. OSS will fail most probably but undefined with respect to when and with what errors.